lib/tokenize.cpp

```diff
@@ -8974,7 +8974,7 @@ void Tokenizer::simplifyFunctionTryCatch()
     for (Token * tok = list.front(); tok; tok = tok->next()) {
         if (!Token::Match(tok, "try {|:"))
             continue;
-        if (!TokenList::isFunctionHead(tok->previous(), "try"))
+        if (!TokenList::isFunctionHead(tok->previous(), "try")) // TODO: this is supposed to a list of characters and not strings
```

Comment:
This was added in da27159d7c704a13790795226803ca764c441d79. Feels like this just works by accident.

CC @dummyunit